### PR TITLE
docs(mkdocs): updated site_url

### PR DESCRIPTION
**Pull request by matt-primrose**
## PR Checklist

- [x] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [x] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?
Added site_url value
Commented out Google Analytics

### CODE REVIEW

**Comment by matt-primrose** [comment]
FYI, I commented out Google Analytics, but we probably still need this.  Need to figure out what the new support for Google Analytics is for the Material theme for mkdocs.

File: mkdocs.yml [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 site_name: 'Open Active Management Technology Cloud Toolkit'
+site_url: 'https://open-amt-cloud-toolkit.github.io/docs'
 repo_url: 'https://github.com/open-amt-cloud-toolkit/open-amt-cloud-toolkit'
 repo_name: 'open-amt-cloud-toolkit/open-amt-cloud-toolkit'
 copyright: 'Copyright &copy; 2021 Intel Corporation'
 site_dir: site/1.4
-google_analytics:
-  - 'UA-185750581-1'
-  - 'auto'
+#google_analytics:
```
Comment (rsdmike) [inline mkdocs.yml:8]: delete these comments

File: mkdocs.yml [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+#  - 'UA-185750581-1'
+#  - 'auto'
 # Configuration
 theme:
   name: 'material'
```
Comment (rsdmike) [inline mkdocs.yml:13]: Add a new property called `gtag: G-00QEDJW2TS` under `theme` and under `analytics`

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 1 above)
Comment (matt-primrose) [reply mkdocs.yml]: done

**Review by kimberlyxdavis — APPROVED** [review]

File: mkdocs.yml (hunk 2 above)
Comment (matt-primrose) [reply mkdocs.yml:10]: I think I got it right now.

**Review by rsdmike — APPROVED** [review]

**Pull request merged**